tests/unit/test_views.py

```diff
@@ -208,15 +208,18 @@ def test_esi_flash_messages():
 class TestSearch:
 
     def _gather_es_queries(self, q):
-        queries = []
-        for field in SEARCH_FIELDS:
-            kw = {"query": q}
-            if field in SEARCH_BOOSTS:
-                kw["boost"] = SEARCH_BOOSTS[field]
-            queries.append(Q("match", **{field: kw}))
+        quoted_strings, unquoted_strings = views.filter_query(q)
+        queries = [
+            views.form_query("phrase", i) for i in quoted_strings
+        ] + [
+            views.form_query("best_fields", i) for i in unquoted_strings
+        ]
+
+        query = Q('bool', must=queries)
```

Comment:
this block is the same as in `views.py`, could we perhaps make it into a function in views that we would import here? Otherwise we are not really testing the implementation.